frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -0,0 +1,297 @@
+import * as _ from 'lodash-es';
+import * as React from 'react';
+
+import { SectionHeading } from '../../utils';
+import { SaveAsDefaultCheckbox } from './alert-manager-receiver-forms';
+
+const SMTP_GLOBAL_FIELDS = [
+  'from',
+  'smarthost',
+  'hello',
+  'auth_username',
+  'auth_password',
+  'auth_identity',
+  'auth_secret',
+  'require_tls',
+];
+const GLOBAL_FIELDS = [...SMTP_GLOBAL_FIELDS]; //TODO follow up PR will add advanced fields
+
+// Ex: converts 'auth_password' to 'emailAuthPassword'
+const getFormFieldName = (propName) =>
+  `email${_.map(_.split(propName, '_'), (part) => _.upperFirst(part)).join('')}`;
+const getFormValue = (formValues, propName) => _.get(formValues, getFormFieldName(propName));
```

Comment:
```suggestion
const getFormValue = (formValues, propName) => formValues[getFormFieldName(propName)];
```

Reply:
types?

Reply:
Unfortunately we have no type safety with this approach :(

Reply:
Yeah, I see what your are saying.  Did a search for 'FromState', do see how the form `type`'s are being done.